proposals/dao/vip_14_arbitrum.ts

```diff
@@ -0,0 +1,102 @@
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+import { assertApproxEq } from '@test/helpers';
+
+/*
+Arbitrum Volt Protocol Improvement Proposal #14
+Description: 
+Deployment Steps
+1. deploy volt system oracle with 0 monthlyChangeRateBasisPoints
+Governance Steps:
+1. Point Oracle Pass Through to new oracle address
+2. Pause DAI PSM
+3. Pause USDC PSM
+*/
+
+/// TODO update this to correct start time
+let startTime;
+
+const monthlyChangeBasisPoints = 0;
```

Comment:
Nit, but maybe make this total explicit and call the var, `ZERO_MONTHLY_CHANGE_BASIS_POINTS`